paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -276,7 +276,8 @@ internal class PaymentOptionsViewModel @Inject constructor(
                 is PaymentSelection.Saved,
                 is PaymentSelection.GooglePay,
                 is PaymentSelection.Link -> processExistingPaymentMethod(paymentSelection)
-                is PaymentSelection.New -> processNewPaymentMethod(paymentSelection)
+                is PaymentSelection.New -> processNewOrExternalPaymentMethod(paymentSelection)
+                is PaymentSelection.ExternalPaymentMethod -> processNewOrExternalPaymentMethod(paymentSelection)
```

Comment:
Can you add a test for this? Maybe something like: https://github.com/stripe/stripe-android/blob/a99222940460a93a16c4097eb23ad82901497a2d/paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsViewModelTest.kt#L102